subgraphs/saucerswap/v2/package.json

```diff
@@ -1,17 +1,15 @@
 {
-  "name": "uniswap-v3-subgraph",
+  "name": "saucerswap-v2-subgraph",
   "version": "1.0.0",
-  "repository": "https://github.com/graphprotocol/uniswap-v3-subgraph",
+  "repository": "https://github.com/hashgraph/hedera-the-graph/tree/main/subgraphs/saucerswap/v2",
   "license": "GPL-3.0-or-later",
   "scripts": {
     "codegen": "graph codegen --output-dir src/types/",
     "build": "graph build",
-    "create-local": "graph create ianlapham/uniswap-v3 --node http://127.0.0.1:8020",
-    "deploy-local": "graph deploy ianlapham/uniswap-v3 --debug --ipfs http://localhost:5001 --node http://127.0.0.1:8020",
-    "deploy": "graph deploy ianlapham/uniswap-v3-subgraph --ipfs https://api.thegraph.com/ipfs/ --node https://api.thegraph.com/deploy/ --debug",
-    "deploy-dev": "graph deploy sommelier/uniswap-v3 --ipfs http://35.197.14.14:5000/ --node http://35.197.14.14:8020/ --debug",
-    "deploy-staging": "graph deploy $THE_GRAPH_GITHUB_USER/$THE_GRAPH_SUBGRAPH_NAME /Uniswap --ipfs https://api.staging.thegraph.com/ipfs/ --node https://api.staging.thegraph.com/deploy/",
-    "watch-local": "graph deploy ianlapham/uniswap-v3 --watch --debug --node http://127.0.0.1:8020/ --ipfs http://localhost:5001"
+    "create": "graph create saucerswap-v2 --node http://127.0.0.1:8020",
+    "deploy-local": "graph deploy saucerswap-v2 --debug --ipfs http://localhost:5001 --node http://127.0.0.1:8020",
+    "deploy-stg": "graph deploy saucerswap-v2 --debug --ipfs https://api.thegraph.com/ipfs/ --node http://127.0.0.1:8020",
+    "remove": "graph remove saucerswap-v2 --node http://127.0.0.1:8020"
```

Comment:
Q: should we have a `watch`?

Reply:
I have not consider it, will investigate and get back 👍

Reply:
What the --watch flag does, it that the `graph-cli` will keep watching the files, and when detecting a change, it will automatically redeploy to the server. 

I don't think we need it, since is useful only during intensive development and test phases, but is not recommended for production environment.